cpython-unix/build-cpython.sh

```diff
@@ -677,6 +677,9 @@ mipsel-unknown-linux-gnu)
 mips64el-unknown-linux-gnuabi64)
     PYTHON_ARCH="mips64el-linux-gnuabi64"
     ;;
+ppc64le-unknown-linux-gnu)
+    PYTHON_ARCH="powerpc64le-linux-gnu"
```

Comment:
i think it should be as well, and i see https://github.com/NixOS/nixpkgs/pull/168083 referencing PEP600 and coming to the same conclusion (setting `pythonHostPlatform` and therefore ` _PYTHON_HOST_PLATFORM` to `ppc64le` : https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/cpython/default.nix#L209) . But... the cpython configure.ac that @ninehills linked does have `powerpc64le-` instead

I see that you're using  `PYTHON_ARCH` to create a symlink. If we change this will this impact that at all or be fine? I was trying to poke around and see where things land as far as directory structure on my dev VM running ppc64le -- but it's currently unavailable so I just have to bother you with questions.

Reply:
I changed PYTHON_ARCH from powerpc64le-linux-gnu to ppc64le-linux-gnu, then built failed:

Github action: [powerpc64le-linux-gnu](https://github.com/ninehills/python-build-standalone/actions/runs/4550383940)

It seems that CPython has hard-coded this value. Unless we patch the CPython source code, it's best to keep the value of PYTHON_ARCH as powerpc64le-linux-gnu.

build error message:

```bash
Traceback (most recent call last):
  File "/home/runner/work/python-build-standalone/python-build-standalone/cpython-unix/build.py", line 1088, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/runner/work/python-build-standalone/python-build-standalone/cpython-unix/build.py", line 1071, in main
    build_cpython(
  File "/home/runner/work/python-build-standalone/python-build-standalone/cpython-unix/build.py", line 803, in build_cpython
    fh.write(build_env.get_output_archive("python"))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/python-build-standalone/python-build-standalone/pythonbuild/buildenv.py", line 116, in get_output_archive
    data = normalize_tar_archive(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/python-build-standalone/python-build-standalone/pythonbuild/utils.py", line 338, in normalize_tar_archive
    filedata = tf.extractfile(ti)
               ^^^^^^^^^^^^^^^^^^
File "/ opt /hostedtoolcache/Python/3.11.2/x64/lib/python3.11/tarfile.py", line 2143, in extractfile
    return self.extractfile(self._find_link_target(tarinfo))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/ opt /hostedtoolcache/Python/3.11.2/x64/lib/python3.11/tarfile.py", line 2452, in _find_link_target
    raise KeyError("linkname %r not found" % linkname)
KeyError: "linkname 'python/install/lib/python3.9/config-3.9d-ppc64le-linux-gnu/libpython3.9d.a' not found"
make: *** [Makefile:278: /home/runner/work/python-build-standalone/python-build-standalone/build/cpython-3.9.16-ppc64le-unknown-linux-gnu-debug.tar] Error 1
Error: Process completed with exit code 2.
```